src/Bicep.LangServer/Snippets/Snippet.cs

```diff
@@ -12,7 +12,7 @@ namespace Bicep.LanguageServer.Snippets
 {
     public sealed class Snippet
     {
-        private static readonly Regex PlaceholderPattern = new Regex(@"\$({(?<index>\d+):(?<name>\w+)}|(?<index>\d+))", RegexOptions.Compiled | RegexOptions.ExplicitCapture);
+        private static readonly Regex PlaceholderPattern = new Regex(@"\$({(?<index>\d+):(?<name>\w+)}|(?<index>\d+)|{(?<index>\d+)\|((?<name>[^,]+)(?<textAfterFirstComma>.*))})", RegexOptions.Compiled | RegexOptions.ExplicitCapture);
```

Comment:
Would you mind adding a comment just to break down the components of this regex and explain what it's looking for?

Reply:
I believe this is the parse the snippet "enum" syntax. Could we change `textAfterFirstComma` to `value` instead?

---
In reply to: [600821745](https://github.com/Azure/bicep/pull/1999#discussion_r600821745) [](ancestors = 600821745)

Reply:
Updated here: https://github.com/Azure/bicep/commit/a2125e0bbad7944b58df6f56fbce04e45c088409

Thanks!